src/pyaro/timeseries/Filter.py

```diff
@@ -855,4 +858,83 @@ def filter_stations(self, stations: dict[str, Station]) -> dict[str, Station]:
         if self._max_altitude is not None:
             stations = {n: s for n, s in stations.items() if (not math.isnan(s["altitude"]) and s["altitude"] <= self._max_altitude) }
         
-        return stations
\ No newline at end of file
+        return stations
+    
+@registered_filter
+class RelativeAltitudeFilter(StationFilter):
+    """
+    Filter class which filters stations based on the relative difference between
+    the station altitude, and the model topography altitude.
+
+    https://github.com/metno/pyaro/issues/39
+    """
+    def __init__(self, topo_file: str | None = None, topo_var: str = "topography", rtol: float = 1):
+        """
+        :param topo_file : A .nc file from which to read model topography data.
```

Comment:
Better call it "gridded topography data", pyaro does not know about any model. (Might also be satellite topography vs obs on a tower...)